modules/sdk-coin-aca/src/aca.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  BaseCoin,
+  BitGoBase,
+  KeyPair,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  SignedTransaction,
+  SignTransactionOptions,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+} from '@bitgo/sdk-core';
+
+export class Aca extends BaseCoin {
+  protected constructor(bitgo: BitGoBase) {
+    super(bitgo);
+  }
+
+  static createInstance(bitgo: BitGoBase): BaseCoin {
+    return new Aca(bitgo);
+  }
+
+  /**
+   * Factor between the coin's base unit and its smallest subdivison
+   */
+  public getBaseFactor(): number {
+    return 1e12;
+  }
+
+  public getChain(): string {
+    return 'aca';
+  }
+
+  public getFamily(): string {
+    return 'aca';
+  }
+
+  public getFullName(): string {
+    return 'Acala';
+  }
```

Comment:
Lets get all this information from statics rather than harcoding. take a look at the dummy draft pr I have opened for reference.